.zuul.yaml

```diff
@@ -0,0 +1,35 @@
+- project:
+    check:
+      jobs:
+        - fog-openstack-unit-test
+        - fog-openstack-integration-test
```

Comment:
There is no integration tests only unit tests done with VCR which corresponds to spec.